### DISCUSSION TITLE
point_0695

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#247] [Roy]
"20200529T184919_20200529T190015_T10SFH"
![image](https://user-images.githubusercontent.com/54947195/147512506-298cf3dd-97c4-4aee-a4b6-efbc9cc34580.png)

"20200115T185721_20200115T185724_T10SFH"
![image](https://user-images.githubusercontent.com/54947195/147512797-5a3ad056-59c0-4f4a-9d53-2abda3efb5ef.png)

"20190629T184921_20190629T185707_T10SFH"
![image](https://user-images.githubusercontent.com/54947195/147512897-d8721e9e-003b-434e-bb67-210639512530.png)
![image](https://user-images.githubusercontent.com/54947195/147512950-9243f7da-54ff-4402-aab5-ced79f72394d.png)

"20190525T184929_20190525T185635_T10SFH"
![image](https://user-images.githubusercontent.com/54947195/147513031-0ac41246-527c-4075-b62c-b2265c8b7418.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200529T184919_20200529T190015_T10SFH"
![Captura](https://user-images.githubusercontent.com/86502862/152838940-75497a4b-36cb-4d29-b245-c3f9c46516a1.JPG)

**Reply by AntoniCesar** [reply]
"20190629T184921_20190629T185707_T10SFH"
![Captura](https://user-images.githubusercontent.com/86502862/152850746-90956259-0b65-46fc-a61e-0836d38eed38.JPG)
